integration-tests/specs/navigation.test.ts

```diff
@@ -0,0 +1,154 @@
+import {driver} from "@wdio/globals";
+
+import {backgroundSessionsEnabled, endSession} from "../helpers/session";
+import {getSpanPayloads} from "../helpers/embrace_server";
+import {EmbraceSpanData} from "../typings/embrace";
+import {
+  commonEmbraceSpanAttributes,
+  embraceSpanDefaults,
+  sortSpanAttributes,
+} from "../helpers/span";
+
+describe("Navigation", () => {
+  it("should record the initially rendered screen", async () => {
```

Comment:
@facostaembrace I was seeing this passing on both Android and iOS which I believe addresses the concern of not capturing the first render, unless there is another test you think could be added?

Reply:
I think it's just fine. We can add more test if we see other weirdness happening